test/components/date-picker.spec.tsx

```diff
@@ -14,7 +15,7 @@ const currentDate = 'CURRENT_DATE';
 const datePickerInputId = 'DATE_PICKER_INPUT';
 const datePickerDropdownId = 'DATE_PICKER_DROPDOWN';
 
-describe('The DatePicker Component', function() {
+describe('The DatePicker Component', () => {
```

Comment:
let's hunt down repeating patterns in this huge test file and refactor them out to methods...